platform/jewel/int-ui/int-ui-standalone/src/main/kotlin/org/jetbrains/jewel/intui/standalone/theme/TextStyles.kt

```diff
@@ -243,3 +409,32 @@ public fun JewelTheme.Companion.createEditorTextStyle(
         hyphens = hyphens,
         textMotion = textMotion,
     )
+
+private fun TextUnit.spValue(): Float {
+    check(isSp) { "Only Sp font sizes are supported" }
+    return value
+}
+
+@Px
+private fun computeInterLineHeightPx(@Px fontSize: Float): Int {
+    val stream = Typography::class.java.classLoader.getResourceAsStream("fonts/inter/Inter-Regular.ttf")
+    val font = Font.createFont(Font.TRUETYPE_FONT, stream).deriveFont(fontSize)
+    return font.computeLineHeightPx()
+}
+
+@Px
+private fun computeJetBrainsMonoLineHeightPx(@Px fontSize: Float): Int {
+    val stream = Typography::class.java.classLoader.getResourceAsStream("fonts/inter/Inter-Regular.ttf")
+    val font = Font.createFont(Font.TRUETYPE_FONT, stream).deriveFont(fontSize)
+    return (font.computeLineHeightPx() * EDITOR_LINE_HEIGHT_FACTOR).roundToInt()
+}
+
+/**
+ * Compensates for some difference in how line height is applied between Skia and Swing. Matches perfectly with the
+ * default editor font.
+ */
+private const val EDITOR_LINE_HEIGHT_FACTOR = 0.87f
```

Comment:
Any reason to be 0.02 bigger than the one in the laf bridge?

Reply:
Eyeballing and pixel peeping :D For some reason it renders slightly shorter in the IDE than outside. Maybe they do some tweaks there, not sure.